jest.config.js

```diff
@@ -4,7 +4,10 @@ module.exports = {
   },
   moduleFileExtensions: ['js', 'jsx'],
   moduleNameMapper: {
-    '^Task1(.*)$': '<rootDir>/src/components/task1$1'
+    '^Task1(.*)$': '<rootDir>/src/components/task1$1',
```

Comment:
Please remove dead code (from task 1) from the codebase.

Reply:
@loginov-rocks fixed